server/src/main/java/org/elasticsearch/common/lucene/search/AutomatonQueries.java

```diff
@@ -129,11 +129,11 @@ public static Automaton toCaseInsensitiveString(String s, int maxDeterminizedSta
         Automaton a = Operations.concatenate(list);
         a = MinimizationOperations.minimize(a, maxDeterminizedStates);
         return a;
- 
-    
+
+
     }
 
-    protected static Automaton toCaseInsensitiveChar(int codepoint, int maxDeterminizedStates) {
+    public static Automaton toCaseInsensitiveChar(int codepoint, int maxDeterminizedStates) {
```

Comment:
Wanted to use this, let me know if this shouldn't be that open. Another option is simpyl copying the logic over I think.

Reply:
LGTM